stable/sentry/values.yaml

```diff
@@ -63,6 +63,33 @@ worker:
   affinity: {}
   # schedulerName:
 
+# Job resources
+jobs:
+  db_init:
```

Comment:
Use camel case

Reply:
I wanted to keep the job resource configuration named the same as the job itself, but I can change it to camel case for consistency's sake.

Reply:
camelcase is the kubernetes convention AFAIK.